migrator/migrations/m_189_to_m_190_plop_add_closed_time_column/migration_impl.go

```diff
@@ -0,0 +1,70 @@
+package m189tom190
+
+import (
+	"github.com/pkg/errors"
+	updatedSchema "github.com/stackrox/rox/migrator/migrations/m_189_to_m_190_plop_add_closed_time_column/schema"
+	"github.com/stackrox/rox/migrator/types"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/pkg/postgres/pgutils"
+	"github.com/stackrox/rox/pkg/utils"
+	"gorm.io/gorm/clause"
+)
+
+var (
+	batchSize = 2000
+	log       = logging.LoggerForModule()
+)
+
+func migrate(database *types.Databases) error {
+	// We are simply promoting a field to a column so the serialized object is unchanged.  Thus, we
+	// have no need to worry about the old schema and can simply perform all our work on the new one.
+	db := database.GormDB
+	pgutils.CreateTableFromModel(database.DBCtx, db, updatedSchema.CreateTableListeningEndpointsStmt)
+	db = db.WithContext(database.DBCtx).Table(updatedSchema.ListeningEndpointsTableName)
+
+	rows, err := db.Rows()
+	if err != nil {
+		return errors.Wrapf(err, "failed to iterate table %s", updatedSchema.ListeningEndpointsTableName)
+	}
+	if rows.Err() != nil {
```

Comment:
idiomatic to do err :=, then we don't need to do rows.Err() repeatedly

Reply:
Not 100% certain what you were getting at here.  But I moved this to the end of the loop.  Since gorm uses pgx underneath, the error may not surface in `rows.Err()` until some later iteration, so `rows.Err()` has to be checked after the loop.